cogs/events.py

```diff
@@ -1,33 +1,36 @@
 import traceback
 
-import discord
-from discord.ext import commands
+from discord.ext import Bot
 
+from discrod import Color, Embed, Message
+from discrod.ext.commands import Cog
 
-class Custom(commands.Cog):
 
-    def __init__(self, client):
-        self.client = client
+class Custom(Cog):
 
-    @commands.Cog.listener()
-    async def on_message(self, message):
-        if message.author.id == self.client.user.id:
+    def __init__(self, bot: Bot) -> None:
+        bot.bot = bot
+
+    @Cog.listener()
+    async def on_message(self, message: Message) -> None:
+        # Check that the message is not from bot account
+        if message.author.bot:
             return
 
         if message.content.lower().startswith("help"):
             await message.channel.send("Hey! Why don't you run the help command with `>>help`")
 
         await self.client.process_commands(message)
 
-    @commands.Cog.listener()
-    async def on_error(self, event, *args, **kwargs):
+    @Cog.listener()
+    async def on_error(self, event, *args, **kwargs) -> None:
         error_message = f"```py\n{traceback.format_exc()}\n```"
         if len(error_message) > 2000:
             async with self.session.post('https://www.hastebin.com/documents', data=error_message) as resp:
                 error_message = 'https://www.hastebin.com/' + (await resp.json())['key']
 
```

Comment:
We have to know and test this part, if it doesn't work, fix it

Reply:
Not sure what can I do about that for now, I have no way to test it as I can't currently start the bot.

Reply:
@ItsDrike i will try to share the token somehow, else i will start the bot, but we need to fix the stackexchange error and do the merging

Reply:
Fixing stackexchange error doesn't really seem like a task for this PR as this is meant for code standardization